ci-operator/config/openshift/cluster-network-operator/openshift-cluster-network-operator-master.yaml

```diff
@@ -196,6 +196,51 @@ tests:
   steps:
     cluster_profile: aws
     workflow: openshift-e2e-aws-single-node
+- as: e2e-network-mtu-migration-ovn-ipv4
+  steps:
+    cluster_profile: gcp
+    env:
+      MTU_DECREASE: "400"
+      MTU_INCREASE: "200"
+      TEST_SKIPS: Alerts shouldn't report any alerts in firing or pending state\|
+        Prometheus when installed on the cluster shouldn't report any alerts in firing
+        state apart from Watchdog and AlertmanagerReceiversNotConfigured
+    test:
+    - chain: network-mtu-migration-decrease
+    - chain: network-mtu-migration-increase
+    - ref: openshift-e2e-test
+    workflow: openshift-e2e-gcp-ovn
+- as: e2e-network-mtu-migration-ovn-ipv6
+  steps:
+    cluster_profile: equinix-ocp-metal
+    env:
+      DEVSCRIPTS_CONFIG: |
+        IP_STACK=v6
+        NETWORK_TYPE=OVNKubernetes
+      MTU_DECREASE: "100"
+      MTU_INCREASE: "50"
+      TEST_SKIPS: Alerts shouldn't report any alerts in firing or pending state\|
+        Prometheus when installed on the cluster shouldn't report any alerts in firing
+        state apart from Watchdog and AlertmanagerReceiversNotConfigured
+    test:
+    - chain: network-mtu-migration-decrease
+    - chain: network-mtu-migration-increase
+    - ref: baremetalds-e2e-test
+    workflow: baremetalds-e2e
+- as: e2e-network-mtu-migration-sdn-ipv4
+  steps:
+    cluster_profile: gcp
+    env:
+      MTU_DECREASE: "400"
+      MTU_INCREASE: "200"
+      TEST_SKIPS: Alerts shouldn't report any alerts in firing or pending state\|
+        Prometheus when installed on the cluster shouldn't report any alerts in firing
+        state apart from Watchdog and AlertmanagerReceiversNotConfigured
+    test:
+    - chain: network-mtu-migration-decrease
+    - chain: network-mtu-migration-increase
+    - ref: openshift-e2e-test
```

Comment:
is this just doing a decrease, then increase, then running tests?

do you want to run tests before or in the middle of the decrease/increase?

Reply:
Yes.

We can only run tests once, so doing that at the end.

Reply:
ok. is it important to run e2e after having decreased MTU as well? if so, we'll need a second job I guess.

Reply:
It might provide some limited value, but then we would need 3 jobs more for a total of 6 if we want to cover everything. The current setup is what I thought it would be reasonable to have initially.

Reply:
sure, whatever seems right to you.